app/assets/images/govuk_publishing_components/chevron-banner/chevron-banner-focus.svg

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg viewBox="0 0 135 109" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+<defs>
+<path id="d" d="M0 0h93v109H0z"/>
+</defs>
+<g fill="#FD0" shape-rendering="geometricPrecision" stroke="#000" stroke-width="2.5px" vector-effect="non-scaling-stroke">
```

Comment:
Crazy thought - instead of having all these different SVGs for hover states, could you use `fill="currentColor"` on some of them instead?

Reply:
I'm not sure that would work with preloading them?